### PR TITLE
[RCF]Fix true colors support

### PR DESCRIPTION
**Pull request by wsdjeg**
@itchyny I am working on true colors support. but this PR is not ready.

### CODE REVIEW

**Comment by wsdjeg** [comment]
@itchyny done, I think it is ok now. can you check it?

File: autoload/screensaver/randomcolor.vim [hunk 1]
```diff
@@ -21,7 +21,9 @@ endfunction
 let s:self = {}
 let s:self.time = -1
 
-if has('gui_running')
+let s:gui = has('gui_running')
+      \ || (has('termguicolors') && &termguicolors == 1)
```
Comment (itchyny) [inline autoload/screensaver/randomcolor.vim]: I think `&termguicolors` is better than `&termguicolors == 1`.

File: autoload/screensaver/randomcolor.vim [hunk 2]
```diff
@@ -31,8 +33,7 @@ else
   let s:updatetime = 6
 endif
 
-let s:gui = has('gui_running')
-let s:is_win32cui = (has('win32') || has('win64')) && !has('gui_running')
+let s:is_win32cui = (has('win32') || has('win64')) && s:gui != 1
```
Comment (itchyny) [inline autoload/screensaver/randomcolor.vim]: `!s:gui` is better than `s:gui != 1`.

Comment (wsdjeg) [inline autoload/screensaver/randomcolor.vim]: yeah.

File: autoload/screensaver/randomcolor.vim (hunk 1 above)
Comment (wsdjeg) [inline autoload/screensaver/randomcolor.vim]: yeah, it is faster.

**Comment by itchyny** [comment]
Thank you.

**Pull request merged**